config/deploy.rb

```diff
@@ -0,0 +1,79 @@
+set :application, "set your application name here"
+set :repository,  "set your repository location here"
+
+# set :scm, :git # You can set :scm explicitly or Capistrano will make an intelligent guess based on known version control directory names
+# Or: `accurev`, `bzr`, `cvs`, `darcs`, `git`, `mercurial`, `perforce`, `subversion` or `none`
+
+role :web, "your web-server here"                          # Your HTTP server, Apache/etc
+role :app, "your app-server here"                          # This may be the same as your `Web` server
+role :db,  "your primary db-server here", :primary => true # This is where Rails migrations will run
```

Comment:
Line is too long. [100/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.